### ISSUE TITLE
HomeAssistant CarPlay Companion not showing devices/entities

### ISSUE THREAD

**Issue by jpatriarca** [home-assistant/iOS#2744]
**iOS device model, version and app version**

Model Name: Apple iPhone 15 Pro Max
Software Version: 17.5
App version: 2024.4

**Home Assistant Core Version**

2024.4.4
**Describe the bug**

When using companion app in CarPlay, when I navigate on **Areas** section, I'm able to see all the sections set up, but when I enter those, I see no entities/devices to control and get information that the section is updating, but no changes occurs.

When accessing to **Control** section and navigate to the specific type of devices/entities, I get the message **No Domains Available**.

But If I use the **Actions** section, I'm able to see any action I have and execute those without any issues.

As it was working until version 2024.02 (at least) but not working since, and as I haven't made any specific changes (that I'm aware) that would bring those errors to the companion, I need your help to debug and identify the root cause. Already reinstalled the app and went through all the setup steps back again and no luck

**To Reproduce**

Trying to access to **Contro** and **Actions** section and no devices/entities are listed

**Expected behavior**

Navigate through the menus would bring devices and entities to take action into by using compaion app for CarPlay

**Screenshots**

![IMG_1256](https://github.com/home-assistant/iOS/assets/19894552/cdcc44dc-c18d-4fe8-a296-f53a4466e436)
![IMG_1250](https://github.com/home-assistant/iOS/assets/19894552/247450a6-b960-43af-9e6c-d105941ab76d)
![IMG_1252](https://github.com/home-assistant/iOS/assets/19894552/d5aa8fb5-722f-44c2-aea3-16156870dbcf)
![IMG_1255](https://github.com/home-assistant/iOS/assets/19894552/9a4f85bc-5caa-4c02-abb8-99eb066ac205)

**Additional context**

Also, with reinstall, the **Actions** were needed to be setup again. Expected that those would be kept in the server during reinstall. Fortunately I just have two of those for now. Not related with this specific issue but something to highlight as something to be reviewed as new feature.

**Comment by gravityrebel** [comment]
Just curious, do you have access to Home Assistant while away from your home network?

**Comment by jpatriarca** [comment]
> Just curious, do you have access to Home Assistant while away from your home network?

Yes, I do. Like specified in my post above, I can use the Actions in place and they work as expected. But the remaining parts, not so much...

**Comment by rafaelcorvalan** [comment]
We you able to fix this issue ? I have exactly the same. Furthermore, the issue seems to be related to the server or how it is configured rather than to the iPhone app. In fact, I have setup 2 servers on my App, and I can see all the devices of one server in the CarPlay configuration, and no device at all from the other server.

**Comment by deepcoder** [comment]
Bump. iOS 18.6 HA iOS App 2025.7.2 TestFlight HA 2025.7.2 Docker

**Comment by IceburgIV** [comment]
Also have similiar issues, saw a post on another forum that said one of your zones might be missing LAT settings, but I have confirmed that not to be the case on mine, and no entities are exposed when going to Settings->Companion App-> CarPlay->Add Item.  I can't add any entities, and search is blank.

I have 2025.10.0, and IOS 26 Public Beta (also tried 26 GA)
I use the Nabu Casa for remote access.

**Comment by ctrolum** [comment]
> Also have similiar issues, saw a post on another forum that said one of your zones might be missing LAT settings, but I have confirmed that not to be the case on mine, and no entities are exposed when going to Settings->Companion App-> CarPlay->Add Item. I can't add any entities, and search is blank.
> 
> I have 2025.10.0, and IOS 26 Public Beta (also tried 26 GA) I use the Nabu Casa for remote access.

Did you found a solution? I have the same issue in the last months…

**Comment by IceburgIV** [comment]
> > Also have similiar issues, saw a post on another forum that said one of your zones might be missing LAT settings, but I have confirmed that not to be the case on mine, and no entities are exposed when going to Settings->Companion App-> CarPlay->Add Item. I can't add any entities, and search is blank.
> > I have 2025.10.0, and IOS 26 Public Beta (also tried 26 GA) I use the Nabu Casa for remote access.
> 
> Did you found a solution? I have the same issue in the last months…

Kind of.  I tried on another phone and it worked fine.  So I determined it was something with the phone, and not the HA environment.  Deleted the server from the app, then deleted the app, and reinstalled it, and it came back.
